pkg/operator/assets/bindata.go

```diff
@@ -41,6 +41,7 @@ import (
 	"strings"
 	"time"
 )
+
```

Comment:
This file is generated, I think you need to skip it or the next person running `hack/update-generated-bindata.sh` will revert.

Reply:
(or, we update that script to run gofmt)

Reply:
I'm just not going to touch it.